pages/css/architecture.md

```diff
@@ -50,46 +56,10 @@ lib/neat.scss
 _overrides.scss
 ```
 
-For the `util`, `typography`, `elements`, and `overrides` files, once they grow
-too large (300 lines or more) in size, split them into their own folder with
-sub files.
```

Comment:
"Too large" is a matter of opinion. I've literally never done this myself and have not seen it on a project in the wild.